### PR TITLE
#162030496 configure project 

### PR DESCRIPTION
**Pull request by MandelaK**
#### What does this PR do?
Adds configuration files to repository

#### Description of Task to be completed?
Configuration files are necessary for our application to work as intended. This PR populates all files with their configurations so we can begin developing our application

#### How should this be manually tested?
This PR adds no testable functionality, but it ensures configuration errors don't occur during application setup.

#### What are the relevant pivotal tracker stories?
#162030496

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: python
+
+python:
+  - "3.6"
+  - "3.6-dev"
```
Comment (kaka-ruto) [inline .travis.yml]: Remove 3.6-dev, let it run only on Python 3.6

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+install:
+  - pip install -r requirements.txt
+  - pip install coveralls
+  - pip install pytest-cov
+
```
Comment (kaka-ruto) [inline .travis.yml:16]: Also install postgres on travis, then set travis to create one test database and run the tests on it

File: README.md [hunk 3]
```diff
@@ -1,2 +1,79 @@
...
+GET /parcels				-		Fetch all parcels in database.
+							Only admin can fetch all parcels in the database
+
+POST /users/parcels			-		Create a delivery order
+							Only logged in users can use this endpoint
+
+
+**To install this repository**
+- clone it - $ git clone https://github.com/MandelaK/SendITAPI-V2
+- create a virtual environment
```
Comment (kaka-ruto) [inline README.md]: Add an example, eg `Create a virtual env, for example virtualenv venv`. Then include the command to start the virtual env, eg `Run source venv/bin/activate to fire up the virtual env`

File: db_config.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import psycopg2
+
+db_url = "dbname='sendit_db' user='postgres' password='postgres' host='localhost' port='5432'"
```
Comment (kaka-ruto) [inline db_config.py]: You never want to store sensitive information, like your database password, in your source code. If your source code is ever seen by anyone, they now have access to your database.
You should conceal/hide your production password in an environment variable.

Also create three databases, a test db, a development db and a production db, best without a password (I prefer you only set up a password for your production db).

Then configure your local machine to work with the development db, like this `export FLASK_ENV=development`

**Review by kaka-ruto — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (MandelaK) [reply README.md]: Does this mean that I track my virtual env or simply that I add instructions on how to create and activate the virtual environment?

File: db_config.py (hunk 4 above)
Comment (MandelaK) [reply db_config.py]: Okay, let me see how I can do this

File: README.md (hunk 3 above)
Comment (kaka-ruto) [reply README.md]: Just give instructions

**Review by kaka-ruto — CHANGES_REQUESTED** [review]

**Pull request merged**